sed/io.py

```diff
@@ -120,7 +124,7 @@ def xarray_to_h5(data: xr.DataArray, faddr: str, mode: str = "w"):
     print("Saving complete!")
 
 
-def h5_to_xarray(faddr: str, mode: str = "r") -> xr.DataArray:
+def load_h5(faddr: str, mode: str = "r") -> xr.DataArray:
```

Comment:
Modify the tutorial accordingly

Reply:
Which tutorial? Tutorial 1 disappeared for some reason.. and there is no call to this in tutorial 2. If you have a copy of tutorial 1 can you re-add it?